sunpy/net/tests/test_fido.py

```diff
@@ -90,13 +98,24 @@ def check_response(query, unifiedresp):
 
 @pytest.mark.remote_data
 def test_save_path():
+    qr = Fido.search(a.Instrument('EVE'), a.Time("2016/10/01", "2016/10/02"), a.Level(0))
+
+    # Test when path is str
     with tempfile.TemporaryDirectory() as target_dir:
-        qr = Fido.search(a.Instrument('EVE'), a.Time("2016/10/01", "2016/10/02"), a.Level(0))
         files = Fido.fetch(qr, path=os.path.join(target_dir, "{instrument}"+os.path.sep+"{level}"))
```

Comment:
`os.path.sep` is not needed there... unless that's what you want to test. `os.path.join` does it for you.
```python
files = Fido.fetch(qr, path=os.path.join(target_dir, "{instrument}", "{level}"))
```